src/Progress/AirtableHandler.php

```diff
@@ -82,6 +83,12 @@ protected function parseProgress(Progress $progress)
             'Snapshot Date' => $progress->getTimestamp()->format(\DateTime::ATOM)
         ];
 
+        if($includeId) {
```

Comment:
Whether or not to include the ID depends on if we have the airtable row ID or not. This can be determined through the airtable_ids database (see comment on ticket, but about the AirtableProgressRow model functions